infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -291,8 +291,6 @@ private void initializationTask() {
                     log.error("detectorTaskScheduler: Recovery failed. Retrying.");
                     continue;
                 }
-                // If recovery succeeds, reconfiguration was successful.
-                triggerSequencerBootstrap(serverContext.copyManagementLayout());
```

Comment:
Maybe now that triggerSequencerBootstrap is moved to runRecoveryReconfiguration the name should change to highlight the new responsibility of this method.